x-pack/plugins/infra/docs/log_structure_helper/new_indexing_strategy_overrides.md

```diff
@@ -0,0 +1,170 @@
+# Managed asset override procedure
+
+## Terms and definitions
+
+- **asset**: An asset means an instance of a stateful Elasticsearch concept such as index, index template, data stream, ingest pipeline.
+- **managed asset**: An managed asset is an asset whose creation, update, and deletion is expected to be performed not by the user, but by a managing piece of code such as the ingest manager or a beat. Index templates are considered managed if the `managed` and `managed_by` properties of the `_meta` property are set. Data streams, indices and and ingest pipelines are considered managed if the corresponding index template is managed.
```

Comment:
Is there a reason there's a need for `managed` **and** `managed_by`, I assume the former is a boolean, could it be inferred from `managed_by`?